app/common/gristUrls.ts

```diff
@@ -95,7 +95,7 @@ export type FormFraming = 'border' | 'minimal';
  *           the original.
  */
 
-export const commonUrls = {
+export const getCommonUrls = () => withAdminDefinedUrls({
   help: getHelpCenterUrl(),
```

Comment:
`GRIST_CUSTOM_COMMON_URLS` should be documented in the README.

For that purpose, I suggest to export an interface in a dedicated file in which each key has a comment for their purpose. Then we may add a link to this file in the README file.